README.md

```diff
@@ -1032,6 +1032,7 @@ To control how the quantitative dimensions *x* and *y* are divided into bins, th
 * **thresholds** - the threshold values; see below
 * **domain** - values outside the domain will be omitted
 * **cumulative** - if positive, each bin will contain all lesser bins
+* **skip** - skip empty bins (default: true)
```

Comment:
How about the name **empty**, where true means to include empty bins, defaulting to false?